docs/content/guides/dagster/asset-tutorial/learning-more.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Learning More | Dagster
+description: You might discover that your needs start to outgrow the basics covered in the tutorial. This page offers a survey of Dagster's full capabilities and where to learn about each of them in docs.
+---
+
+# Learning More
```

Comment:
My initial impressions of this were:

1. Wow, this is really thorough!
2. This is a bit much to read for having just finished a tutorial with a lot of steps.
3. Am I going to want to click to another page after going through all that?
4. What do the terms in the headings mean to me as a brand-new user?

For 1-3, perhaps consider condensing this down to just the necessities and adding it to the existing **Conclusion** section at the end of the tutorial - that's currently [here](https://docs.dagster.io/guides/dagster/asset-tutorial/testing-assets#conclusion). We can always move that section to another page if a different step will be last in the new tutorial.

My thinking here is going to yet *another* page after completing a lengthy-ish tutorial might feel like a chore. If we add a section with a few short bullet points that's right at the end where the user already is, they don't have to go to another page to find ideas for what's next.

**For 4 (terms in headings)**, what would the headings/descriptions look like if we lead with what those features accomplish? Ex: **IO managers** > **Customize asset storage using IO managers**

Reply:
I do have some thoughts on the actual copy in here as well, but I'll hold off until we nail down the approach to this page/info overall before adding those.